src/fediblockhole/__init__.py

```diff
@@ -721,6 +721,40 @@ def save_domain_block_audit_to_file(
         for key, value in sorted_list:
             writer.writerow(value)
 
+def resolve_replacements(endpoints: list[dict]) -> list[dict]:
+    """Resolve any replacement tokens in the list of endpoints"""
+    resolved = []
+    for item in endpoints:
+        item = dict(**item)
+        if 'token' in item and 'token_env_var' in item:
+            log.warning(f"Both `token` and `token_env_var` have been provided; using the explicit token for {item.get("domain", "the entry")}")
+            # we take the token that's explicitly stated, even if there's also an environment
+            # variable set. Delete the token_env_var key
+            del item['token_env_var']
+
+        elif 'token' in item:
+            ...
+
+        elif 'token_env_var' in item:
+            value = os.getenv(item['token_env_var'])
```

Comment:
Below `os.environ.get()` is used. Why the difference here? `os.getenv()` uses `os.environ`, I believe, so functionally it should achieve the same outcome, but can we be consistent?

Reply:
Sure, we'll go with the os.environ.get()